wp-content/themes/eighshift-task/src/blocks/layout/footer/_footer-navigation.scss

```diff
@@ -0,0 +1,29 @@
+$footer-navigation: (
+  footer-nav-spacing: $base-gutter-size,
+  footer-nav-text: $base-nobel-color,
+  footer-nav-highlight-text: $base-primary-color,
+);
+
+.footer-navigation {
+  @extend %list-reset;
+  display: flex;
+  align-items: center;
+
+  &__item {
+    padding-left: map-get($footer-navigation, 'footer-nav-spacing');
```

Comment:
use map-get-strict insted of map-get. it has additional check on error and it will help you a lot if you have an error